src/nft/components/explore/Explore.css.ts

```diff
@@ -0,0 +1,120 @@
+import { style } from '@vanilla-extract/css'
+import { breakpoints, sprinkles } from 'nft/css/sprinkles.css'
+
+/* Override for BodyWrapper properties */
+export const fullWidth = style([
+  sprinkles({
+    left: '1/2',
+    right: '1/2',
+    width: 'full',
+    maxWidth: 'full',
+    position: 'absolute',
+  }),
+  {
+    top: '72px',
+    marginLeft: '-50vw',
+    marginRight: '-50vw',
+  },
+])
+
+export const bannerWrap = style([
+  sprinkles({
+    position: 'relative',
+    overflow: 'hidden',
+    height: '386',
+  }),
+  {
+    backgroundPosition: 'center',
+    backgroundSize: 'cover',
+  },
+])
+
+export const bannerOverlay = style([
+  {
+    opacity: '0.7',
+    height: '386px',
+  },
+  sprinkles({
+    position: 'absolute',
+    zIndex: '0',
+    width: 'full',
+    backgroundColor: 'grey900',
+    left: '0',
+    top: '0',
+  }),
+])
+
+export const bannerContent = sprinkles({
+  position: 'relative',
+  zIndex: '1',
+})
+
+export const collectionName = style([
+  sprinkles({
+    textAlign: 'left',
+    overflow: 'hidden',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    display: 'inline-block',
+    color: 'explicitWhite',
+  }),
+  {
+    maxWidth: 'calc(100% - 80px)',
+  },
+])
+
+export const collectionDetails = style([
+  sprinkles({
+    width: 'full',
+  }),
+  {
+    '@media': {
+      [`screen and (min-width: ${breakpoints.tabletL}px)`]: {
+        width: '40%',
+      },
+    },
+  },
+])
+
+export const volumeRank = style([
+  sprinkles({
+    paddingTop: '8',
+    paddingBottom: '8',
+    paddingRight: '16',
+    paddingLeft: '16',
+    color: 'green200',
+  }),
+  {
+    background: 'rgba(92, 254, 157, 0.12)',
+    borderRadius: '64px',
+  },
+])
+
+export const exploreCollection = style([
+  {
+    width: '174px',
```

Comment:
We do mod 4 px values in almost all cases--if this is spec'd to 174 it's likely a goof from design, should double check this.